app/src/test/java/com/example/forzenbook/data/RepositoryUnitTest.kt

```diff
@@ -0,0 +1,151 @@
+        coEvery { errorService.getToken("", "") } returns Response.error(
+            400,
+            ResponseBody.create(null, "")
+        )
+
+        var repository = LoginRepositoryImpl(dao, succeedsService)
+        runBlocking {
+            try {
+                Assert.assertEquals(repository.getToken("", "").token, testString)
+            } catch (e: Exception) {
+                fail()
+            }
+        }
+        repository = LoginRepositoryImpl(dao, nullTokenService)
+        runBlocking {
+            try {
+                repository.getToken("", "")
+                fail()
+            } catch (e: Exception) {
+                Assert.assertEquals(e.message, "Failed to get Token")
+                // TODO when FA-84 is accepted this section may need to be updated
+            }
+        }
+        repository = LoginRepositoryImpl(dao, errorService)
+        runBlocking {
+            try {
+                repository.getToken("", "")
+                fail()
+            } catch (e: Exception) {
+                Assert.assertEquals(e.message, "Failed to get Token")
+                // TODO when FA-84 is accepted this section may need to be updated
+            }
+        }
+    }
+
+    @Test
+    fun `CreateAccount Repository test, checking success case, and making sure throws are correct`() {
+        val succeedsService = mockk<CreateAccountService>()
+        val userExistsService = mockk<CreateAccountService>()
+        val errorService = mockk<CreateAccountService>()
+
+        coEvery {
+            succeedsService.createUser(
+                "",
+                Date.valueOf("2020-01-01"),
+                "",
+                "",
+                ""
+            )
+        } returns Response.success(null)
+        coEvery {
+            userExistsService.createUser(
+                "",
+                Date.valueOf("2020-01-01"),
+                "",
+                "",
+                ""
+            )
+        } returns Response.error(
+            409,
+            ResponseBody.create(null, "")
+        )
+        coEvery {
+            errorService.createUser(
+                "",
+                Date.valueOf("2020-01-01"),
+                "",
+                "",
+                ""
+            )
+        } returns Response.error(
+            400,
+            ResponseBody.create(null, "")
+        )
+
+        var repository = CreateAccountRepositoryImpl(succeedsService)
+        runBlocking {
+            try {
+                repository.createUser("", "", "2020-01-01", "", "")
+            } catch (e: Exception) {
+                fail()
+            }
+        }
+        repository = CreateAccountRepositoryImpl(userExistsService)
+        runBlocking {
+            try {
+                repository.createUser("", "", "2020-01-01", "", "")
+            } catch (_: AccountException) {
+            } catch (e: Exception) {
```

Comment:
Catch the errors and assert there are the type you are expecting instead of just assuming they go that branch. Also, in your try blocks, put `fail()` after your initial statements to ensure the test _must_ fail